### PR TITLE
docs(installation): simplify run.sh commands

### PR DESCRIPTION
**Pull request by oguzkaganozt**
## Description

Simplify run commands for docker containers

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by xmfcx** [comment]
From here:
- https://github.com/autowarefoundation/autoware/pull/4738#issuecomment-2125232206

> image
> 
> @oguzkaganozt I was looking into https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/docker-installation/#prerequisites and noticed the [setup-dev-env.sh](https://github.com/autowarefoundation/autoware/blob/main/setup-dev-env.sh) doesn't have any "docker" references, so is this documentation wrong or outdated?

**Comment by oguzkaganozt** [comment]
> From here:
> 
> * [feat(ci,docker): change `vcs import` to `COPY src` autoware#4738 (comment)](https://github.com/autowarefoundation/autoware/pull/4738#issuecomment-2125232206)
> 
> > image
> > @oguzkaganozt I was looking into https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/docker-installation/#prerequisites and noticed the [setup-dev-env.sh](https://github.com/autowarefoundation/autoware/blob/main/setup-dev-env.sh) doesn't have any "docker" references, so is this documentation wrong or outdated?

@xmfcx [Here](https://github.com/autowarefoundation/autoware/blob/ab8bbf56ecdacbabae89f56f4bc8b6ea1d7bf0df/setup-dev-env.sh#L82) is the setting of target playbook which sets it as docker when you call` setup_dev_env.sh docker`

File: docs/installation/autoware/docker-installation.md [hunk 1]
```diff
@@ -59,15 +69,15 @@ Inside the container, you can run the Autoware tutorials by following these link
 
 [Rosbag Replay Simulation](../../tutorials/ad-hoc-simulation/rosbag-replay-simulation.md).
 
-### Development setup
+### Development Environment
```
Comment (xmfcx) [inline docs/installation/autoware/docker-installation.md:72]:
```suggestion
### Development environment
```

**Comment by oguzkaganozt** [comment]
@youtalk @xmfcx

**Review by youtalk — APPROVED** [review]
LGTM

**Pull request merged**